boards/stm32f4discovery/include/periph_conf.h

```diff
@@ -29,6 +29,30 @@
 extern "C" {
 #endif
 
+/**
+ * @name    DMA streams configuration
+ * @{
+ */
+#ifdef MODULE_PERIPH_DMA
+static const dma_conf_t dma_config[] = {
+    { .stream = 11 },   /* DMA2 Stream 3 - SPI1_TX */
+    { .stream = 10 },   /* DMA2 Stream 2 - SPI1_RX */
+    { .stream = 4 },    /* DMA1 Stream 4 - SPI2_TX */
+    { .stream = 3 },    /* DMA1 Stream 3 - SPI2_RX */
+};
+
+#define DMA_0_ISR           isr_dma2_stream3
+#define DMA_1_ISR           isr_dma2_stream2
+#define DMA_2_ISR           isr_dma1_stream4
+#define DMA_3_ISR           isr_dma1_stream3
+#define DMA_4_ISR           isr_dma1_stream5
```

Comment:
The 2 last ISR defines have to be removed as well ;)

Reply:
Arrrgh, thanks, removed.